src/references/damageable_objects.md

```diff
@@ -0,0 +1,97 @@
+---
+id: damageable_reference
+name: Damageable Objects Reference
+title: Damageable Objects Reference
+tags:
+    - Reference
+---
+
+# Damageable Objects
+
+## Overview
+
+The **Damageable Object** is a **Core Object** that implements the **Damageable** interface and has properties, events, and functions for objects which can take damage and die.
+
+The **Damageable Object** streamlines damage and destroying objects. Children of a **Damageable Object** that are hit by a projectile (or another damage source), will look for an ancestor that is a **Damageable** and automatically deduct the damage from health of the **Damageable**. You have control over what happens when the health of the **Damageable Object** reaches 0.
```

Comment:
You have control > Creators can control ... through the [DamageableObject type in the Core Lua API](link to the API page)

Just to be *absolutely clear* that this can't be done through, say, the Properties panel